src/components/LineChart/LineChart.jsx

```diff
@@ -371,12 +380,14 @@ const LineChart = createClass({
 			yAxisTitleColor,
 			yAxisMin,
 			yAxisTooltipFormatter,
+			yAxisTooltipDataFormatter,
```

Comment:
i think you could get rid of some of the conditional ternary stuff below by making them defaults here:

```
yAxisFormatter: yScale.tickFormat(),
yAxisTooltipFormatter: yAxisFormatter,
yAxisTooltipDataFormatter: yAxisTooltipFormatter,
```